cpp/dockerfiles/Dockerfile.arm64-linux-gcc

```diff
@@ -1,6 +1,8 @@
+# Unused & not maintained. This probably won't work without changes.
+
 FROM aztecprotocol/crosstool-ng-arm64:latest
 WORKDIR /usr/src/barretenberg/cpp
 COPY . .
-RUN mkdir build && cd build && cmake -DTOOLCHAIN=arm64-linux-gcc .. && cmake --build . --parallel
+RUN cmake --toolchain ./cmake/toolchains/aarch64-linux.cmake --preset gcc && cmake --build --preset gcc
```

Comment:
Why not use a preset here?

Reply:
It's using the gcc preset and adding the cross-compiling toolchain file. We could have presets for all cross-compilation, but that's usually a matrix, so it'd drastically increase the number of presets we have to maintain.